src/themes/playful.css

```
@@ -244,22 +260,22 @@
   /* Border widths are designed to scale according to a single base value.
      * The base value is intended for calculations and is not used by components directly. */
   --wa-border-width-base: 0.125;
```

```suggestion
  --wa-border-width-base: calc(2 / 18);
```

If we want to ensure whole pixel values for borders at 100% zoom, one trick I like is using a `calc()` function to divide the desired pixel values by the root font size. That prevents us from having to mess with potentially messy decimals.

Since it looks like we have 2px, 3px, and 4px borders used throughout our components with this theme, we may also want to adjust the calculations below to multiply by `1rem`, `1.5rem`, and `2rem`.